src/machines/child_workflow_state_machine.rs

```diff
@@ -54,6 +54,12 @@ fsm! {
     Started --(ChildWorkflowExecutionTimedOut(i32), shared on_child_workflow_execution_timed_out) --> TimedOut;
     Started --(ChildWorkflowExecutionCancelled, shared on_child_workflow_execution_cancelled) --> Cancelled;
     Started --(ChildWorkflowExecutionTerminated, shared on_child_workflow_execution_terminated) --> Terminated;
+
+    // Ignore any spurious cancellations after resolution
+    Cancelled --(Cancel) --> Cancelled;
```

Comment:
Hmm.. just verifying, I think we only use this machine to cancel before sending a command, no?
There's the external WF machine that does cancellation commands.

Reply:
Also, double cancel is probably a lang SDK bug (at least in TS).